src/test/groovy/org/prebid/server/functional/util/privacy/gpp/TcfEuV2Consent.groovy

```diff
@@ -0,0 +1,32 @@
+package org.prebid.server.functional.util.privacy.gpp
+
+import com.iab.gpp.encoder.field.TcfEuV2Field
+
+class TcfEuV2Consent extends GppConsent {
+
+    protected TcfEuV2Consent(Regions regions, def fieldValues) {
+        super(regions, fieldValues)
+    }
+
+    static class Builder extends GppConsent.Builder {
```

Comment:
Please add all the same builder method as in `TcfConsent.groovy `(e.g `setVersion()` and all other that we actually use)